client/components/data/query-reader-follows/index.jsx

```diff
@@ -0,0 +1,25 @@
+/**
+ * External dependencies
+ */
+import { Component } from 'react';
+import { connect } from 'react-redux';
+
+/**
+ * Internal dependencies
+ */
+import { requestFollows } from 'state/reader/follows/actions';
+
+class QueryReaderFollows extends Component {
```

Comment:
we should make a HoC that takes the action to dispatch on mount and returns a component that does that. This pattern is crazy common.